src/views/SafetyStock.vue

```diff
@@ -8,7 +8,7 @@
     </ion-header>
 
     <ion-content>
-      <main v-if="ruleGroup.ruleGroupId">
+      <main v-if="ruleGroup.ruleGroupId && rules.length">
         <ScheduleRuleItem v-if="rules.length" />
```

Comment:
We can remove this if check of rules.length as we have added the same on the main tag, handle the same on all the places.